app/models.py

```diff
@@ -853,21 +864,31 @@ def website_send_to(self):
 
     def new_addr(self):
         """
-        Replace original email by reply_email. 2 possible formats:
+        Replace original email by reply_email. Possible formats:
         - first@example.com by SimpleLogin <reply_email> OR
-        - First Last - first at example.com <reply_email>
+        - First Last - first at example.com <reply_email> OR
+        - First Last - first@example.com <reply_email>
         And return new address with RFC 2047 format
 
         `new_email` is a special reply address
         """
         user = self.user
-        if user and user.use_via_format_for_sender:
+        if (
+            not user
+            or not SenderFormatEnum.has_value(user.sender_format)
+            or user.sender_format == SenderFormatEnum.VIA.value
+        ):
             new_name = f"{self.website_email} via SimpleLogin"
-        else:
+        elif user.sender_format == SenderFormatEnum.AT.value:
             name = self.name or ""
             new_name = (
                 name + (" - " if name else "") + self.website_email.replace("@", " at ")
             ).strip()
+        elif user.sender_format == SenderFormatEnum.C_AT.value:
```

Comment:
It misses a final `else` here and *in theory* `new_name` can be undefined at the end of the method.
I wonder if we can slightly refactor the code into 2 cases here: `user is not None` and `user is None` . Within the first case, we construct the `new_name` according to user preference.

Reply:
That's essentially what's already there: (It's essentially a switch statement, with the default at the top)
```
if (no user) or (unknown format code) or (format = VIA)
---
elif (format = AT)
---
elif (format = C_AT)
```